.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -0,0 +1,460 @@
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: check
+
+  - name: test
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: test
+        vars:
+          debug: 'hadron*,mongo*'
+
+  - name: test-electron
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: test-electron
+        vars:
+          debug: 'hadron*,mongo*'
+
+  - name: test-connectivity
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: clone
+      - func: test-connectivity
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  - name: test-csfle
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: test-csfle
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  - name: e2e-coverage
+    tags: ['run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: e2e-coverage
+        vars:
+          debug: 'compass-e2e-tests*,electron*,hadron*,mongo*'
+
+  - name: generate-vulnerability-report
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: generate-vulnerability-report
+        vars:
+          debug: 'compass*,electron*,hadron*,mongo*'
+
+  # Publish happens in one go to make sure we are not creating multiple github
+  # releases in parallel
+  - name: publish
+    # Does a dry-run: doesn't actually publishes, but checks that all files
+    # exist and we are not trying to do a "broken" publish
+    tags: ['run-on-pr']
+    depends_on:
+      - name: '.required-for-publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: mongodb-compass
+      - func: get-all-artifacts
+      - func: publish
+
+  - name: publish-packages-next
+    tags: []
+    depends_on:
+      - name: '.required-for-publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+      - func: publish-packages-next
+
+  - name: publish-dev-release-info
+    tags: []
+    depends_on:
+      - name: 'publish'
+        variant: '*'
+    commands:
+      - func: prepare
+      - func: publish-dev-release-info
+
+  - name: test-web-sandbox-chrome
+    tags: ['required-for-publish', 'run-on-pr']
+    commands:
+      - func: prepare
+      - func: install
+      - func: bootstrap
+        vars:
+          scope: 'compass-e2e-tests'
+      - func: apply-compass-target-expansion
+        vars:
+          compass_distribution: compass
+      - func: test-web-sandbox
+        vars:
+          mongodb_version: 'latest-alpha-enterprise'
```

Comment:
Is that right? Should this be the same as testing platforms and use latest stable?

Reply:
Yeah I also think this should be latest stable enterprise.

Reply:
good catch, this was like that already, not sure why, we can probably use latest stable here as well. Now i'm also thinking if we should repeat the server matrix for web on chrome, if there is significant branching.

Reply:
I'll follow up on this as well